microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -140,51 +138,54 @@ After you verified that the animation isn't performing well, the next step is to
 
 1. When no events are selected, the **Summary** panel shows you a breakdown of activity.  The page spent most of the time rendering.  Since performance is the art of doing less work, your goal is to reduce the amount of time spent doing rendering work.
 
-   ![The Summary panel](./index-images/evaluate-performance-performance-summary-tab.png)
+   ![The Summary panel](./index-images/summary-tab.png)
 
 1. Expand the **Main** section.  DevTools shows you a flame chart of activity on the main thread, over time.  The x-axis represents the recording, over time.  Each bar represents an event.  A wider bar means that event took longer.  The y-axis represents the call stack.  When events are stacked on top of each other, it means the upper events caused the lower events.
 
-   ![The Main section](./index-images/evaluate-performance-performance-main.png)
+   ![The Main section](./index-images/main.png)
 
 1. There is a lot of data in the recording.  To zoom into a portion of the recording, click and drag in the **Overview** area toward the top of the **Performance** tool.  The **Overview** area includes the **CPU** and **NET** charts (indicated on the right).  The **Main** section and **Summary** panel only display information for the selected portion of the recording.
 
-   ![Zoom into a section](./index-images/evaluate-performance-performance-main-zoomed.png)
+   ![Zoom into a section](./index-images/main-zoomed.png)
 
    Another way to change the selected area is to put focus on the **Main** section, click the background or an event, and then press:
    * `W` to zoom in, `S` to zoom out.
    * `A` to move selection left, `D` to move selection right.
 
-1. Select an **Animation Frame Fired** event.  When a red triangle is displayed at the top right of an event, it's a warning that there might be an issue related to the event.
+1. Click an **Animation Frame Fired** event.  When a red triangle is displayed at the top right of an event, it's a warning that there might be an issue related to the event.
 
    The **Animation Frame Fired** event occurs whenever a [requestAnimationFrame() callback](https://developer.mozilla.org/docs/Web/API/window/requestAnimationFrame) is run.
```

Comment:
```suggestion
1. Click an **Animation Frame Fired** event.

   When a red triangle is displayed at the top right of an event, it's a warning that there might be an issue related to the event.  The **Animation Frame Fired** event occurs whenever a [requestAnimationFrame() callback](https://developer.mozilla.org/docs/Web/API/window/requestAnimationFrame) is run.
```
compact version of pattern:
1. Do X using UI controls.  Y happens:

   Screenshot

verbose version of pattern, used here:

1. Do X using UI controls.

   Y happens.  A couple sentences about Y happening:

   Screenshot

the idea is to separate the user action from the result, when there's substantial explanatory info about the result.